.ci/env/openblas.sh

```diff
@@ -163,7 +163,10 @@ pushd "${blas_src_dir}"
         NO_FORTRAN=1
         USE_OPENMP=0
         USE_THREAD=0
-        USE_LOCKING=1)
+        USE_LOCKING=1
+        DYNAMIC_ARCH=1
+        DYNAMIC_LIST="NEHALEM HASWELL"
```

Comment:
Does it still throw the same 'illegal instruction' error if you let it use the default "sapphire rapids" instead of manually listing these kernels here?

Reply:
It does not fail with the default build settings, yes. But I thought maybe to reduce the build time by listing the specific kernels.